libcxx/include/string_view

```diff
@@ -689,6 +689,9 @@ private:
 
   const value_type* __data_;
   size_type __size_;
+
+  template<class, class, class>
+  friend class basic_string;
```

Comment:
Let's add `#include <__fwd/string.h>`. It's already transitively included via `__fwd/string_view.h`, but let's be IWYU clean.

Reply:
Ah, that explains one of the diagnostics I was getting! We don't need to include that header though, since the friend declaration doubles as the class declaration (see https://godbolt.org/z/bj1PMh44b). Do you still want me to add it?

Reply:
I think I would, in case we declare it with attributes or something like that. It would be nice for the "canonical" declaration to always be included. It's already included transitively right now anyway, but I'd rather be intentional about it.